internal/network/manifests_generator.go

```diff
@@ -141,12 +141,14 @@ spec:
             After=network.target
 
             [Service]
-            ExecStart=/usr/sbin/dnsmasq -k
+            ExecStart=/usr/sbin/dnsmasq -k {{.DNSMASQ_ADDITIONAL_PARAMS}}
 
             [Install]
             WantedBy=multi-user.target
 `
 
+const additionalDNSMasqParams = "-a 0.0.0.0 -a ::"
```

Comment:
my understanding is that ``::`` is just one address whereas ``::/0`` includes any address in the space of ipv6 addresses
any way, it's not critical. we'll just make sure to test the ipv6 case

Reply:
Yep, it is 0.0.0.0 in ipv6
Don't see why not to have same behavior as we have now